VotingApplication/VotingApplication.Web/Scripts/Controllers/ManageExpiryController.js

```diff
@@ -0,0 +1,84 @@
+﻿/// <reference path="../Services/ManageService.js" />
+(function () {
+    angular
+        .module('GVA.Creation')
+        .controller('ManageExpiryController', ManageOptionController);
+
+    ManageOptionController.$inject = ['$scope', '$routeParams', '$location', 'ManageService'];
+
+    function ManageOptionController($scope, $routeParams, $location, ManageService) {
+
+        $scope.poll = ManageService.poll;
+        $scope.manageId = $routeParams.manageId;
+        $scope.timeOption = null;
+
+        $scope.updatePoll = updatePollDetails;
+        $scope.return = navigateToManagePage;
+        $scope.formatExpiry = formatExpiry;
+        $scope.removeExpiry = removeExpiry;
+        $scope.setDate = setDate;
+        $scope.timeOffset = timeOffset;
+
+        activate();
+        
+        function activate() {
+            ManageService.registerPollObserver(function () {
+                $scope.poll = ManageService.poll;
+            });
+        }
+
+        function formatExpiry() {
+            if ($scope.poll && $scope.poll.ExpiryDate){
+                return moment($scope.poll.ExpiryDate).format("dddd, MMMM Do YYYY, HH:mm");
+            }
+            return 'Never';
+        }
+
+        function navigateToManagePage() {
+            $location.path('Manage/' + $scope.manageId);
+        };
+
+        function updatePollDetails() {
+            ManageService.updatePoll($routeParams.manageId, $scope.poll, updatePollSuccessCallback);
+        };
+
+        function updatePollSuccessCallback() {
+            ManageService.getPoll($scope.manageId);
```

Comment:
I'm guessing this call is to refresh the poll and notify all the observers via the ManageService? If so, we probably need to take another look at the ManageService, as it's not obvious at all.